app/bundles/CoreBundle/Loader/ParameterLoader.php

```diff
@@ -77,7 +76,6 @@ public function loadIntoEnvironment(): void
         EnvVars\SessionEnvVars::load($this->parameterBag, $defaultParameters, $envVariables);
         EnvVars\SiteUrlEnvVars::load($this->parameterBag, $defaultParameters, $envVariables);
         EnvVars\TwigEnvVars::load($this->parameterBag, $defaultParameters, $envVariables);
-        MessengerEnvLoader::load($this->parameterBag, $defaultParameters, $envVariables);
```

Comment:
Why did you remove this? We need it load default Messenger config in case the DSN string is wrong

Reply:
please elaborate, what do you mean wrong?

Reply:
There is no reason to remove . Another PR use it

> MessengerEnvLoader::load($this->parameterBag, $defaultParameters, $envVariables);

Reply:
if you mean #12509 he deleted it too.